kotsadm/pkg/handlers/troubleshoot.go

```diff
@@ -538,6 +539,29 @@ func makeKurlCollectors() []*v1beta1.Collect {
 	return rookCollectors
 }
 
+func makeVeleroCollectors() []*v1beta1.Collect {
+	selectors := []string{
+		"component=velero",
+		"app.kubernetes.io/name=velero",
+	}
+
+	collectors := []*v1beta1.Collect{}
+	for _, selector := range selectors {
+		collectors = append(collectors, &v1beta1.Collect{
+			Logs: &v1beta1.Logs{
+				CollectorMeta: v1beta1.CollectorMeta{
+					CollectorName: "velero",
+				},
+				Name:      "velero",
+				Selector:  []string{selector},
+				Namespace: "velero",
```

Comment:
is there a function in the snapshots package that finds this?

Reply:
good idea. updated.